packages/agoric-cli/lib/chain-config.js

```diff
@@ -124,6 +124,21 @@ export function finishCosmosGenesis({ genesisJson, exportedGenesisJson }) {
   genesis.app_state.mint.params.inflation_rate_change = '0.0';
   genesis.app_state.mint.params.inflation_min = '0.0';
 
+  /*
+		FIXME: Allow these to be configurable.
```

Comment:
Do these need to be addressed before merge?

Reply:
As described above, I will land a separate PR for that.  Our faucet account is the same across all our deployed chains, and this code only affects genesis.jsons that we create, so I'd do it as a cleanup step.